cannon/mipsevm/evm_test.go

```diff
@@ -36,9 +37,19 @@ func testContractsSetup(t *testing.T) (*Contracts, *Addresses) {
 }
 
 func SourceMapTracer(t *testing.T, contracts *Contracts, addrs *Addresses) vm.EVMLogger {
-	mipsSrcMap, err := contracts.MIPS.SourceMap([]string{"../../packages/contracts-bedrock/contracts/cannon/MIPS.sol"})
+	sources := bindings.Sources
+	contractsDir := "../../packages/contracts-bedrock"
+	for i, source := range sources {
+		// Add relative path to contracts directory if the source is not
+		// already relativized.
+		if !strings.HasPrefix(source, "..") {
+			sources[i] = path.Join(contractsDir, source)
+		}
+	}
```

Comment:
This would probably be a good candidate for a separate function like `relativizeSources(sources []string) []string` but nbd